proto/frequenz/api/microgrid/common.proto

```diff
@@ -145,91 +148,94 @@ message AC {
   // AC metrics of a single phase.
   message ACPhase {
     // The AC voltage between the line and the neutral wire.
-    // In Volts (V).
+    // In Volt (V).
     Metric voltage = 1;
 
     // AC current.
     // +ve current means consumption, away from the grid.
     // -ve current means supply into the grid.
-    // In Amperes (A).
+    // In Ampere (A).
     Metric current = 2;
 
     // AC active power.
     // +ve power means consumption, away from the grid.
     // -ve power means supply into the grid.
-    // In Watts (W).
+    // In Watt (W).
     Metric power_active = 3;
 
     // AC reactive power.
     // +ve power means inductive (leading).
     // -ve power means capacitive (lagging).
-    // In VArh.
+    // In Volt-Ampere reactive hour (VArh).
     Metric power_reactive = 4;
 
     // The total apparent energy. A Positive value represents the net apparent
     // energy supplied to the grid connection, and vice versa.
-    // In VAh.
+    // In Volt-Ampere-hour (VAh).
     Metric energy_apparent = 5;
 
     // The total active energy counters for the underlying component's
     // consumption and supply.
-    // In Watthours (Wh).
+    // In Watt-hour (Wh).
     ActiveEnergy energy_active = 6;
 
     // The total reactive energy counters for the underlying component's
     // capacitive and inductive energy values.
-    // In VArh.
+    // In Volt-Ampere reactive hour (VArh).
+    // FIXME: ReactiveEnergy says it's in VAh
```

Comment:
Need feedback here.

Reply:
Reactive energy is in `VArh`.

Here are some units for context: 
Active power: Watts (W)
Active energy: Watt-hours (Wh) (Note the time component)
Reactive power: Volt-ampere-reactive (VAr) (Note the 'r' for reactive)
Reactive energy: Volt-ampere-reactive-hours (VArh)
Apparent power: Volt-amperes (VA) (Note the 'r' for reactive)
Apparent energy: Volt-ampere hours (VAh)

Reply:
What I mean is in the definition of the `ReactiveEnergy` message, it says is in VAh:

https://github.com/frequenz-floss/frequenz-api-microgrid/blob/0ffff8010646d9d824367021be9a5bc99aee7b5a/proto/frequenz/api/microgrid/common.proto#L112-L127

Reply:
I updated `ReactiveEnergy` to use `VArh`, I guess that should be correct.

Reply:
Oh, right. `VArh` is correct.